database/models/organization.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class Organization extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  };
```

Comment:
Pasa lo mismo que en migrations

Reply:
Aquí no entiendo a lo que te refieres. Podrías explicarmelo? según los requerimientos no tiene ninguna relación con otra tabla.

Reply:
creo que el hace referencia a los not null y eso, pero pues acá como tal no es obligatorio